ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -320,6 +311,17 @@ post_provision_config_nodes() {
         dnf -y config-manager --add-repo="${repo_url}"
         disable_gpg_check "$repo_url"
     done
+
+    if ! "${cmd[@]}" --exclude golang-*.daos.*; then
```

Comment:
I suppose, by moving this down here to make the `dnf upgrade` upgrade any pre-existing golang with the one from the PR, we need to remove the `--exclude golang-*.daos.*` as that is excluding golang from the upgrade.

```suggestion
    if ! "${cmd[@]}"; then
```